### ISSUE TITLE
Can't run --library option

### ISSUE THREAD

**Issue by mricaud** [ndw/xmlcalabash1#136]
It  seems this is a bug in last Calabash-1.0.16-95 when running a specified step in a library (option -l ou --library in calabash command line).
# COMMAND

```
java -Xms512m -Xmx1024m -jar calabash.jar -D -b igs=http://www.igs-cp.fr/custom -l library.xpl -s igs:run -i source=input.xml
```
# XPL : library.xpl

```
    <?xml version="1.0"?>
    <p:library   xmlns:p="http://www.w3.org/ns/xproc"
                 xmlns:igs="http://www.igs-cp.fr/custom"
                 version="1.0">
      <p:pipeline type="igs:run">
        <p:identity/>
      </p:pipeline>
    </p:library>
```
# XML : input.xml

```
    <?xml version="1.0" encoding="UTF-8"?>
    <TESTS>
      <TEST>1</TEST>
      <TEST>2</TEST>
      <TEST>3</TEST>
    </TESTS>
```

What's happening : 

The pipeline never ends up, sometimes a java error (parsing) is thrown (when trying to write something in the console under windows...)

**Comment by ndw** [comment]
When you're building a pipeline up on the command line, you have to put the sources before the step they apply to:

java -Xms512m -Xmx1024m -jar calabash.jar -D -b igs=http://www.igs-cp.fr/custom -l library.xpl -i source=input.xml -s igs:run

But looking at this code, I'm not sure there isn't more trouble brewing. Let me know how it works for you in practice.

**Comment by mricaud** [comment]
I've tried your solution with Calabash1.0.16 but still have the same problem :(

**Comment by ndw** [comment]
Really? Odd. I'm pretty sure it worked for me.

**Comment by mricaud** [comment]
Yes really, I've tried one more time today by copy/past your line, same result : I get a message with the implicit pipeline and then nothing else, waiting...
I ran it on windows seven, java ""1.7.0_07", with saxonHe and also with saxonPe (site licence).
That's strange.
